### ISSUE TITLE
3D GUIs

### ISSUE THREAD

**Issue by karai17** [excessive/DOMinatrix#10]
While this may not exactly be important for LOVE proper, I am working on a LOVE3D project and would like for this to support both 2D and 3D GUIs. More thought needs to be put into 3D GUIs, so if you have any ideas, toss them here. :)

**Comment by adrix89** [comment]
Oh god no!
Why do people keep want 3D from a 2D engine.
It has all been downhill since they implemented meshes.

3D is complicated, make it a separate project if you want 3D.

**Comment by karai17** [comment]
Have you seen our LOVE3D project? www.LandonManning.com

**Comment by shakesoda** [comment]
Yeah, this is going to accommodate 3D GUIs regardless. This also isn't only going to be for LOVE, it will be for HATE, which treats 3D as a first-class citizen.

Also, what? All downhill since adding meshes? Are you insane? Meshes are fantastically useful.

**Comment by adrix89** [comment]
The 3D insanity has taken ahold of you!
Quick! Exorcise it!
You have become fascinated about making engines not games, worse you are using an engine to make an engine!

**Comment by shakesoda** [comment]
it's what happens when your framework doesn't do enough and the next best thing does a hundred times too much.

**Comment by karai17** [comment]
Yeah, it's either tinker with expanding LOVE or dive head first into Unreal Engine 4. UE4 is fantastic, but it scares me. I simply don't know enough about the deep and dirty of game development to be comfortable attempting to learn to use UE4, whereas with LOVE it's fairly simple and allows me to learn new things at a pace where I will remember them.

**Comment by adrix89** [comment]
UE4 is not the answer



Unity is.


With its thousands of plugins that make the game for you.

**Comment by karai17** [comment]
Cleaned out some very off topic discussion. Please keep these issues on topic. If you have a legitimate concern, feel free to voice it.
